src/screens/login.js

```diff
@@ -1,27 +1,44 @@
-import React , { useState } from "react";
-import { Text, View, TextInput, Button } from "react-native";
-import CheckBox from '@react-native-community/checkbox';
+import React, { useState } from "react";
+import { Text, View, TouchableOpacity, Image } from "react-native";
+import { SafeAreaView } from "react-native-safe-area-context"
+import { InputPassword, FormButton, InputEmail } from '../components/index'
 import Styles from '../screens/style/common_style'
 
+
+
 export const login = (props) => {
+  const {navigation} = props
+  return (
+
```

Comment:
Remove the unnecessary spaces.

Reply:
We have done what you asked and we deleted the fonts that we are not using.